change "lurkable" to "Discoverable" on get guild preview route (#6195)
hammerandchisel/discord-api-docs: docs/resources/Guild.md

## Get Guild Preview % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/preview

Returns the [guild preview](#DOCS_RESOURCES_GUILD/guild-preview-object) object for the given id. 

> If the user is not in the guild, then the guild must have the "DISCOVERABLE" [feature](#DOCS_RESOURCES_GUILD/guild-object-guild-features).

Reviewer: ```suggestion
If the user is not in the guild, then the guild must be [discoverable](#DOCS_RESOURCES_GUILD/guild-object-guild-features).
```
Understand where your edit is coming from, but want to stick with saying a guild is discoverable, rather than has the "DISCOVERABLE" feature.

> If the user is not in the guild, then the guild must be have the "DISCOVERABLE" [feature](#DOCS_RESOURCES_GUILD/guild-object-guild-features).

Reviewer 2: ```suggestion
If the user is not in the guild, then the guild must have the "DISCOVERABLE" [feature](#DOCS_RESOURCES_GUILD/guild-object-guild-features).
```
